docs/developers/bitesize/IoActor/IoActor.md

```diff
@@ -0,0 +1,36 @@
+# IoActor: basic concepts
+
+* **Word count:** 225
+
+## Actor Hierarchy
+
+IO subsystem consists of the following actors.
+
+![IoActor hierarchy](IoActor.png)
```

Comment:
Could you show the client actor on the actor diagram?

Reply:
Do you mean showing all concrete actors, which send messages to `IoActorProxy`? Or just an abstract "Client Actor" box, in order to show that messages are sent to `IoActorProxy`?

Reply:
The latter - just an abstract client is fine (unless it's a **really** small set (ie only 1 or 2), in which case it might be possible to list them)